Maple2.Server.Game/Model/Field/Actor/FieldNpc.cs

```diff
@@ -210,6 +215,14 @@ public class FieldNpc : Actor<Npc> {
             foreach (string tag in Value.Metadata.Basic.MainTags) {
                 player.Session.ConditionUpdate(ConditionType.npc_race, codeString: tag);
             }
+		}
+	}
+	
+    public void SetAi(string name) {
+        AiMetadata? metadata;
+
+        if (Field.AiMetadata.TryGet(name, out metadata)) {
+            AiMetadata = metadata;
         }
```

Comment:
this works?
```suggestion
        Field.AiMetadata.TryGet(name, out AiMetadata)
```

Reply:
This may accidentally make AiMetadata null

Reply:
Isn't already null? what's the difference

Reply:
I assume this function exists to update the Ai of an existing Npc, so it may not be null and then become null

Reply:
I can add a warning for when that happens, but npcs getting null ai metadata also isn't undesirable if there's a debugging command added for changing it. Pretty sure null metadata is something that should be expected since not all npcs even have the field set.
`11200003.xml: <aiInfo path="" />`

Reply:
I think you should use a special case for that (e.g. empty string is passed in) rather than allowing it to be nulled whenever a script cannot be found.